tests/Setup/ReminderFactory.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Tests\Setup;
+
+use App\Reminder;
+
+class ReminderFactory
```

Comment:
Again, consistency is key:
This is a good idea and has been used lately by many devs in the ecosystem - however we're using regular factories in our project, so this would be a deviation.